supabase/migrations/20240505115913_releases.sql

```diff
@@ -5,7 +5,11 @@ CREATE TABLE public.releases (
      release_notes TEXT,
      created_by UUID REFERENCES public.users ON DELETE SET NULL,
      created_at TIMESTAMP WITH TIME ZONE NOT NULL,
-     updated_at TIMESTAMP WITH TIME ZONE NOT NULL
+     updated_at TIMESTAMP WITH TIME ZONE NOT NULL,
+     git_tag_name Text NOT NULL,
+     github_release_created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+     github_release_published_at TIMESTAMP WITH TIME ZONE NOT NULL,
+     CONSTRAINT unique_git_tag_per_project UNIQUE (project_id, git_tag_name)
```

Comment:
I realized that it can be still easily checked if a Git tag was already used in the project. Since the whole project is connected to one repository, there's no need to check the GitHub repository owner or repository slug. You can simply use the following constraint:

```
 CONSTRAINT unique_git_tag_per_project UNIQUE (project_id, git_tag_name)
```

NOTE:
_Once the github refactoring will be done, you wont'be able to switch your project to different github repository (once you create at least one release). If your project would use different repo, you will need to archive the current one and create a new one._

Reply:
Nice. But do we really need the `github_release_created_at` and `github_release_published_at` fields?
(Btw, isn't it supposed to be `github_release_created_at` and `github_release_updated_at`?)

If somebody updates a release in GitHub, then both `updated_at` and `github_release_updated_at` will be updated, right?
If somebody updates a release via Release Manager, well, again, both fields will have to be updated, no?.. 🤔 

If somebody needs finer details on whether it was release in Release Manager who triggered updates or release in GitHub, well he or she can take a look at timestamps in GitHub.

Reply:
Oh, and btw, in the scenario when we update release in Release Manager and propagate it to GitHub Releases, this will trigger the webhook and.. infinite loop?.. 💀 🔁 💣

Reply:
> Nice. But do we really need the github_release_created_at and github_release_published_at fields?

You are right. We dont really need it now. Let's drop it.

We could then simply have 

```
type GithubRelease struct {
  GitTagName string
  URL url.URL
}  
```

Reply:
Or maybe even better, simply

```
GitTagName string
GithubReleaseURL url.URL
```

Reply:
Yes, that's what I meant by "flattening". 👍 

I would prefer it.
Of course, service model doesn't have to reflect the db model 1-to-1, but from practical perspective, it might be much easier to keep it similar for now, and do the splitting later on both svc and db layers.